main.nf

```diff
@@ -36,6 +36,9 @@ include { PHANTOM_PEAKS
           PPQT_PROCESS
           MULTIQC                  } from "./modules/local/qc.nf"
 
+
+contrastsheet = params.contrastsheet ?: "/assets/contrast_test.ymls"
```

Comment:
should this be

```suggestion
contrastsheet = params.contrastsheet ?: "${projectDir}/assets/contrast_test.ymls"
```

to make sure it uses the project directory as root, instead of the root of the file system?